hybridCarboniteMetadataService/src/main/scala/cromwell/services/metadata/hybridcarbonite/HybridCarboniteConfig.scala

```diff
@@ -12,25 +15,51 @@ import net.ceedubs.ficus.Ficus._
 
 import scala.concurrent.Await
 import scala.concurrent.duration._
+import scala.language.postfixOps
 import scala.util.Try
 
-final case class HybridCarboniteConfig(enabled: Boolean, pathBuilders: PathBuilders, bucket: String) {
+final case class HybridCarboniteConfig(enabled: Boolean, pathBuilders: PathBuilders, bucket: String, freezeScanConfig: HybridCarboniteFreezeScanConfig) {
   def makePath(workflowId: WorkflowId)= PathFactory.buildPath(HybridCarboniteConfig.pathForWorkflow(workflowId, bucket), pathBuilders)
 }
 
+final case class HybridCarboniteFreezeScanConfig(initialInterval: FiniteDuration = 5 seconds, maxInterval: FiniteDuration = 5 minutes, multiplier: Double = 1.1)
+
 object HybridCarboniteConfig {
 
   def pathForWorkflow(id: WorkflowId, bucket: String) = s"gs://$bucket/$id/$id.json"
   
   def parseConfig(carboniterConfig: Config)(implicit system: ActorSystem): Checked[HybridCarboniteConfig] = {
     val enable = carboniterConfig.as[Option[Boolean]]("enabled").getOrElse(false)
 
+    def freezeScanConfig: Checked[HybridCarboniteFreezeScanConfig] = {
+      if (carboniterConfig.hasPath("freeze-scan")) {
+        val freeze = carboniterConfig.getConfig("freeze-scan")
+
+        val initialInterval = Try { freeze.getOrElse("initial-interval", 5 seconds) } toErrorOr
+        val maxInterval = Try { freeze.getOrElse("max-interval", default = 5 minutes) } toErrorOr
+        val multiplier = Try { freeze.getOrElse("multiplier", 1.1) } toErrorOr
+
+        val errorOrFreezeScanConfig = (initialInterval, maxInterval, multiplier) mapN {
+          case (i, m, x) => HybridCarboniteFreezeScanConfig(i, m, x)
+        } contextualizeErrors "parse Carboniter 'freeze-scan' stanza"
+
+        for {
+          freezeScanConfig <- errorOrFreezeScanConfig.toEither
+          _ <- if (freezeScanConfig.maxInterval > freezeScanConfig.initialInterval) "".validNelCheck else "'max-interval' must be greater than or equal to 'initial-interval' in Carboniter 'freeze-scan' stanza".invalidNelCheck
```

Comment:
TOL does `ficus` interact well enough with `Int Refined Positive` to give you a free "parse as a positive int" method from config?

Reply:
Uh, this comment is on the wrong sanity check line, but you get the idea

Reply:
we want > 1 tho